pkgs/tools/security/pinentry/default.nix

```diff
@@ -57,7 +60,6 @@ pinentryMkDerivation rec {
   dontWrapQtApps = true;
 
   patches = [
-    ./autoconf-ar.patch
```

Comment:
Actually, without this patch, cross-compilation (`pkgsCross.aarch64-multiplatform.pinentry`) fails:

```
/nix/store/pmrhk324fkidrm5ffd5jckb21s9zys6r-bash-4.4-p23/bin/bash: ar: command not found
```

cc @nspin who added it in https://github.com/NixOS/nixpkgs/pull/68889